.github/workflows/ci.yaml

```diff
@@ -167,3 +167,18 @@ jobs:
           file: ${{ matrix.connector }}/Dockerfile
           push: true # See 'if' above
           tags: ghcr.io/estuary/${{ matrix.connector }}:dev
+
+      - name: Push ${{ matrix.connector }} image with version tag
+        if: ${{ github.event_name == 'push' }}
+        run: |
+          export IMAGE_VERSION=$(grep VERSION ${{ matrix.connector }}/Dockerfile | cut -d = -f 2)
```

Comment:
Perhaps `grep '^LABEL VERSION'` to avoid accidentally triggering on some `FOOBAR_VERSION` string that might be mentioned somewhere?